src/index.ts

```diff
@@ -79,9 +77,9 @@ export class Spectral {
   public getRules(dataFormat?: string): IRuleEntry[] {
     const rules = [];
 
-    for (const name in this._rules) {
-      if (!this._rules.hasOwnProperty(name)) continue;
-      const { rule, format, apply } = this._rules[name];
+    for (const name in this._rulesByIndex) {
+      if (!this._rulesByIndex.hasOwnProperty(name)) continue;
+      const { rule, format, apply } = this._rulesByIndex[name];
 
       if (!dataFormat || format.indexOf(dataFormat) !== -1) {
```

Comment:
Don't really like matching formats by using indexOf, I think it should be an exact match so `format === dataFormat`